src/org/labkey/test/tests/FileAttachmentColumnTest.java

```diff
@@ -100,7 +100,7 @@ private void doSetup()
 
     private void createList()
     {
-        beginAt("/project/" + getProjectName() +"/"+ FOLDER_NAME + "/begin.view?");
+        beginAt(getProjectName() + "/" + FOLDER_NAME + "/begin.view");
```

Comment:
Missed the controller.
```suggestion
        beginAt(getProjectName() + "/" + FOLDER_NAME + "/project-begin.view");
```